README.md

```diff
@@ -80,12 +81,95 @@ No inputs are required for this action.
+| `common-mvn-flags`        | Maven flags for all subsequent mvn calls                                                                                   | `--batch-mode --no-transfer-progress --errors --fail-at-end --show-version -Dmaven.test.redirectTestOutputToFile=false` |
+| `maven-opts`              | Parameters for the JVM started for subsequent mvn calls                                                                     | `-Xmx1536m -Xms128m`                                                |
+
+### Outputs
+
+| Output             | Description                                                                                    |
+|--------------------|------------------------------------------------------------------------------------------------|
+| `build-number`     | The current build number. Also set as environment variable `BUILD_NUMBER`                     |
+| `snapshot-version` | The project version set in the pom.xml (before replacement). Also set as environment variable `CURRENT_VERSION` |
+| `project-version`  | The project version with build number (after replacement). Also set as environment variable `PROJECT_VERSION` |
+
+### Environment Variables Set
+
+After running this action, the following environment variables are available:
+
+- `BUILD_NUMBER`: The current build number
+- `SNAPSHOT_VERSION`: The original project version from pom.xml
+- `PROJECT_VERSION`: The project version with build number appended
+- `ARTIFACTORY_USERNAME`: Username for Artifactory authentication
+- `ARTIFACTORY_ACCESS_TOKEN`: Access token for Artifactory authentication
+- `ARTIFACTORY_PASSWORD`: Deprecated alias for `ARTIFACTORY_ACCESS_TOKEN`
+- `ARTIFACTORY_ACCESS_USERNAME`: Deprecated alias for `ARTIFACTORY_USERNAME`
+- `MAVEN_OPTS`: JVM options for Maven execution
+- `SONARSOURCE_REPOSITORY_URL`: URL for SonarSource Artifactory repository
+- `BASH_ENV`: Path to bash aliases file with Maven command alias
+
+### Features
+
+- **Automatic Build Number Management**: Integrates with `get-build-number` action for consistent build numbering
+- **Project Version Configuration**: Automatically updates Maven project version with build number using format `<MAJOR>.<MINOR>.<PATCH>.<BUILD_NUMBER>`
+- **Artifactory Integration**: Sets up authentication and repository configuration for SonarSource Artifactory
+- **Maven Settings Management**: Provides pre-configured `settings.xml` with SonarSource repositories and authentication
+- **Maven Local Repository Caching**: Automatically caches `~/.m2/repository` for faster builds
+- **Common Maven Flags**: Sets up convenient Maven command alias with common flags for consistent execution
+- **Repository Visibility Detection**: Automatically configures appropriate Artifactory repositories based on repository visibility (public/private)
```

Comment:
this duplicates https://github.com/SonarSource/ci-github-actions/pull/93/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R88-R94 no?